ACID/acid_tbc.sql

```diff
@@ -18287,12 +18292,13 @@ INSERT INTO `creature_ai_scripts` (`id`,`creature_id`,`event_type`,`event_invers
 ('1894207','18942','2','3','100','0','15','0','0','0','25','0','0','0','1','-47','0','0','0','0','0','0','Innalia - Flee at 15% HP (Phase 2)'),
 ('1894208','18942','7','0','100','0','0','0','0','0','22','1','0','0','40','1','0','0','0','0','0','0','Innalia - Set Phase 1 and Set Melee Weapon Model on Evade'),
 -- Bonechewer Scavenger
-('1895201','18952','4','0','20','0','0','0','0','0','1','-765','0','0','22','1','0','0','0','0','0','0','Bonechewer Scavenger - Say and Set Phase 1 on Aggro'),
-('1895202','18952','0','5','100','0','1000','3000','0','0','11','13398','1','0','0','0','0','0','0','0','0','0','Bonechewer Scavenger - Cast Throw Wrench (Phase 1)'),
-('1895203','18952','9','5','100','1','0','5','6000','9000','11','34113','1','0','0','0','0','0','0','0','0','0','Bonechewer Scavenger - Cast Bonechewer Bite (Phase 1)'),
-('1895204','18952','2','0','100','0','15','0','0','0','22','1','0','0','0','0','0','0','0','0','0','0','Bonechewer Scavenger - Set Phase 2 at 15% HP'),
-('1895205','18952','2','3','100','0','15','0','0','0','25','0','0','0','1','-47','0','0','0','0','0','0','Bonechewer Scavenger - Flee at 15% HP (Phase 2)'),
-('1895206','18952','7','0','100','0','0','0','0','0','22','1','0','0','0','0','0','0','0','0','0','0','Bonechewer Scavenger - Set Phase 1 on Evade'),
+('1895201','18952','4','0','20','0','0','0','0','0','22','-1','0','0','0','0','0','0','0','0','0','0','Bonechewer Scavenger - Set Phase 1 on Aggro'),
```

Comment:
Working on it. It also sets the wrong phase and the inverse phase masks are wrong.